src/geom-pole-of-inaccessibility.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef OSM2PGSQL_GEOM_POLE_OF_INACCESSIBILITY_HPP
+#define OSM2PGSQL_GEOM_POLE_OF_INACCESSIBILITY_HPP
+
+/**
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This file is part of osm2pgsql (https://osm2pgsql.org/).
+ *
+ * Copyright (C) 2006-2022 by the osm2pgsql developer community.
+ * For a full list of authors see the git log.
+ */
+
+#include "geom.hpp"
+
+namespace geom {
+
+/**
+ * Finding the "pole of inaccessibility", the most distant internal point from
+ * the polygon outline, or center of the maximum inscribed circle.
+ *
+ * \param polygon The input polygon
+ * \param precision Used as cutoff in the recursive algorithm. A minimum cutoff
+ *        is also set at max(width, height) of the polygo envelope / 1000.0.
```

Comment:
```suggestion
 *        is also set at max(width, height) of the polygon envelope / 1000.0.
```

Reply:
Thanks for the typos found. They are fixed now.